pkg/services/folder/model.go

```diff
@@ -149,6 +150,12 @@ type GetFolderQuery struct {
 	SignedInUser identity.Requester `json:"-"`
 }
 
+type GetFoldersQuery struct {
+	OrgID        int64
+	UIDs         []string
+	WithFullpath bool
```

Comment:
Nit: Shall we rename it to `WithFullPath` since we have three separate words in the name?

Reply:
Actually I see that we are using "fullpath" as a single word throughout the code. Is that intentional?